src/test/java/com/kainos/ea/backend/services/JobFamilyServiceTest.java

```diff
@@ -28,4 +39,25 @@ void when_getJobFamiliesByCapabilityName_expect_RepositoryCalledPassback() {
         Mockito.verify(jobFamilyRepository).findByCapabilityName("");
         assertEquals(jobFamily, results);
     }
+
+    @Test
+    public void when_updateJobFamilyName_expect_RepositoryCalledPassbackFind() {
+        Mockito.when(jobFamilyRepository.findById("Name")).thenReturn(Optional.of(jobFamily));
+
+        jobFamilyService.updateJobFamilyName("Name", "Name");
+
+        Mockito.verify(jobFamilyRepository).findById("Name");
+    }
+
+    @Test
+    public void when_updateJobFamilyName_expect_RepositoryCalledPassbackdelete() {
```

Comment:
Please change the 'd' in delete to upper-case

Reply:
Done!